api/views/andela_centres.py

```diff
@@ -36,12 +38,48 @@ class AndelaCentreViewset(ModelViewSet):
     permission_classes = [IsAuthenticated, IsAdminUser]
     authentication_classes = [FirebaseTokenAuthentication]
 
+    def create(self, request, *args, **kwargs):
+        if request.user.is_superuser:
+            return super().create(request, *args, **kwargs)
+        else:
+            raise PermissionDenied("Only a super user can create a center")
+
     def destroy(self, request, *args, **kwargs):
         instance = self.get_object()
         self.perform_destroy(instance)
         data = {"detail": "Deleted Successfully"}
         return Response(data=data, status=status.HTTP_204_NO_CONTENT)
 
+    @action(
+        detail=True,
+        permission_classes=[IsAuthenticated],
```

Comment:
It's ok to use IsAdmin here as one of the permission_classes

Reply:
DONE